modules/optimization/test/tests/optimizationreporter/point_loads/tests

```diff
@@ -59,7 +59,7 @@
       type = CSVDiff
       input = main_auto_adjoint.i
       absent_out = 'type: ilu'
-      cli_args = 'Outputs/file_base=main_out -nl0_ksp_view -adjoint_ksp_view'
+      cli_args = 'Outputs/file_base=main_out -nl0_ksp_view -adjoint_ksp_view -ksp_view'
```

Comment:
Can you add a comment above this making sure that -ksp_view is not removed. The reason this came undetected is that -ksp_view was taken out of the test during a PR.

Reply:
I created this issue https://github.com/idaholab/moose/issues/30188 for us to add a test to verify that nondefault input file preconditioners are being used for each system.  So I approve of everything in the current PR.